lib/improver/tests/nbhood/test_squareneighbourhood.py

```diff
@@ -159,26 +141,92 @@ def test_for_multiple_realizations_and_times(self):
         self.assertArrayAlmostEqual(result.data, data)
 
 
-class Test_run(IrisTest):
-
-    """Test the run method on the SquareNeighbourhood class."""
+class Test_mean_over_neighbourhood(IrisTest):
 
-    RADIUS_IN_KM = 10
+    """Test for calculating mean value in neighbourhood"""
 
-    def test_basic(self):
-        """Test that a cube with correct data is produced by the run method"""
+    def setUp(self):
+        """Set up cube and expected results for tests."""
         data = np.array([[1., 2., 3., 4., 5.],
```

Comment:
I think it is worth adding a note to say this came from adding up an array of 1s with a 0 in the middle.